_unused_topics/cluster-logging-exported-fields-docker.adoc

```diff
@@ -5,7 +5,7 @@
 [id="cluster-logging-exported-fields-container_{context}"]
 = Container exported fields
 
-These are the Docker fields exported by the {product-title} cluster logging available for searching from Elasticsearch and Kibana.
+These are the Docker fields exported by the OpenShift Logging available for searching from Elasticsearch and Kibana.
```

Comment:
In this example, "the OpenShift Logging" sounds odd to me. Suggest dropping "the"